packages/page-staking/src/Nomination/EraToTime.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useApi} from '@polkadot/react-hooks';
+import { useTranslation} from '@polkadot/app-accounts/translate';
+import { BareProps} from "@polkadot/react-api/types";
+import { BlockToTime} from "@polkadot/react-query";
+import { formatNumber } from '@polkadot/util';
+
+interface EraToTimeInterface extends BareProps{
+  showBlocks?: boolean;
+  showDays?: boolean;
+}
+
+function EraToTime({ className, style, showBlocks, showDays }: EraToTimeInterface) {
+  const { api } = useApi();
+  const { t } = useTranslation();
+  const bondedDuration = api.consts.staking.bondingDuration;
+  const epochDuration = api.consts.babe.epochDuration;
+  const genesisSlot = api.query.babe.genesisSlot;
```

Comment:
We really don't want to be doing this, we are not a Babe-only UI. Use the derives. Even better, there is a hook now in staking that converts eras to blocks/time.